src/scanner.ts

```diff
@@ -96,7 +96,9 @@ export function doScan() {
             }
 
             // mark the batch done in a few seconds
-            setTimeout(finishBatch, 5000)
+            setTimeout(() => {
+              finishBatch(batchId)
+            }, 5000)
```

Comment:
So weird how making something take longer makes more sense to slow human expectation.